### PR TITLE
build/configs/rtl8720e/loadable_apps: Update defconfig

### PR DESCRIPTION
**Pull request by thapav**
This patch:
1) Updated partition info to boot tp1e board
2) Removes unnecessary configs

Signed-off-by: Vidisha Thapa <thapa.v@samsung.com>

### CODE REVIEW

**Review by sunghan-chang — APPROVED** [review]

File: build/configs/rtl8720e/loadable_apps/defconfig [hunk 1]
```diff
@@ -460,10 +460,10 @@ CONFIG_SERIAL=y
 # CONFIG_SERIAL_CONSOLE is not set
 # CONFIG_16550_UART is not set
 # CONFIG_ARCH_HAVE_UART is not set
-CONFIG_ARCH_HAVE_UART0=y
+# CONFIG_ARCH_HAVE_UART0 is not set
```
Comment (sunghan-chang) [inline build/configs/rtl8720e/loadable_apps/defconfig:463]: Could you disable below as well?
```
CONFIG_UART0_RXBUFSIZE=1024
CONFIG_UART0_TXBUFSIZE=1024
CONFIG_UART0_BAUD=115200
CONFIG_UART0_BITS=8
CONFIG_UART0_PARITY=0
CONFIG_UART0_2STOP=0
.....
```

Comment (sunghan-chang) [reply build/configs/rtl8720e/loadable_apps/defconfig:463]: same as uart2 and 3

Comment (thapav) [reply build/configs/rtl8720e/loadable_apps/defconfig:463]: updated

File: build/configs/rtl8720e/loadable_apps/defconfig [hunk 2]
```diff
@@ -549,29 +534,13 @@ CONFIG_UART2_2STOP=0
...
-# CONFIG_UART4_IFLOWCONTROL is not set
-# CONFIG_UART4_OFLOWCONTROL is not set
 
 #
 # UART1 Configuration
 #
 
 #
-# UART2 Configuration
+# UART4 Configuration
```
Comment (kishore-sn) [inline build/configs/rtl8720e/loadable_apps/defconfig:543]: I am not sure why these lines have come here.  Either we can move them above to the proper places or we could remove them.

Comment (thapav) [reply build/configs/rtl8720e/loadable_apps/defconfig:543]: I have updated as per the .config
May be its due to the generic Kconfigs. We could modify them in a new PR

**Comment by sunghan-chang** [comment]
`arch/arm/src/amebalite: Update unutized UART ports in the Kconfig`
`unutized` looks a typo.

File: build/configs/rtl8720e/loadable_apps/defconfig (hunk 2 above)
Comment (thapav) [reply build/configs/rtl8720e/loadable_apps/defconfig]: updated

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**